slrealizer/utils/script.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Can this be removed, now that `slrealizer/make_lens_source.py` exists?

Reply:
Oh yes. Was Jenny's and forgot to remove.